lib/packages/docutils/rst.nim

```diff
@@ -1212,7 +1213,7 @@ proc parseInline(p: var RstParser, father: PRstNode) =
       father.add(n)
     elif isInlineMarkupStart(p, "`"):
       var n = newRstNode(rnInterpretedText)
-      parseUntil(p, n, "`", true)
+      parseUntil(p, n, "`", false) # bug #17260
```

Comment:
This call is also used for parsing references (they will change their type to `rnRef` or `rnHyperLink` in the `parsePostfix` proc). But according to the spec backslash **does** escaping in the references.

Reply:
can you give me a minimal example (eg nim doc comment or rst sample)?
so i know how to trigger this; then i can probably add a flag to distinguish

Reply:
An example from the spec:


> If the reference text happens to end with angle-bracketed text that is not a URI or hyperlink reference, at least one angle-bracket needs to be backslash-escaped or an escaped space should follow. For example, here are three references to titles describing a tag:
```
See `HTML Element: \<a>`_, `HTML Element: <b\> `_, and
`HTML Element: <c>\ `_.
```

Reply:
However I tried it and it is not working currently anyway.

```
HTML Element: <a>
-----------------

See `HTML Element: \<a>`_
```

So it can be left for future work.

Reply:
can you create an issue `misc rst` (similar to `misc testament` https://github.com/nim-lang/Nim/issues/16521) as a meta-issue to track things like this?
(scope: to track rst issues that aren't major and dont' require a dedicated issue)
(I could also do it but i think you have the most experience with rst)

Reply:
Good idea. Done: #17340